examples/wearable/UIComponents/contents/list/virtual/virtuallist.js

```diff
@@ -39,13 +39,6 @@
 				});
 			}
 		});
-		if (pageId.indexOf("marquee") > -1) {
```

Comment:
Since this part of code is no longer needed (listview supports marquee by default) please consider removing the whole "1 line with  marquee" page.